### DISCUSSION TITLE
any fix?

### DISCUSSION THREAD

**Discussion by vytska69** [ROM-builders/temporary#9810] [General]
hey. Im trying to build crdroid for mi439 with new trees, but facing syncing issues. Looked in my lm, its normal. Maybe its cirrus or crdroid manifest issue?

**Comment by Apon77** [comment]
> hey. Im trying to build crdroid for mi439 with new trees, but facing syncing issues. Looked in my lm, its normal. Maybe its cirrus or crdroid manifest issue?

build link?

**Comment by vytska69** [comment]
https://cirrus-ci.com/build/5544791197876224

**Reply by hsx02** [reply]
You have to change mi439 to Mi439 
Like, lineage_Mi439.mk
Directory should be changed

device/xiaomi/Mi439
vendor/xiaomi/Mi439

**Reply by vytska69** [reply]
no no, check sync error, its probably cirrus issue. And with building was normal in old tree by cafebabe

**Reply by hsx02** [reply]
Before that, check cafebabe source you will understand
I already succed Mithorium build

**Comment by Apon77** [comment]
> hey. Im trying to build crdroid for mi439 with new trees, but facing syncing issues. Looked in my lm, its normal. Maybe its cirrus or crdroid manifest issue?

check #failed instruction, test things locally first, i think its issue from your side, may be because of case sensitive, not sure though

**Reply by vytska69** [reply]
ok, i'll do repo sync locally and see
